ng-dev/utils/config.ts

```diff
@@ -23,6 +23,8 @@ export interface GitClientConfig {
   useSsh?: boolean;
   /** Whether the specified repository is private. */
   private?: boolean;
+  /** Main branch name for the repository. */
+  mainBranchName?: string;
```

Comment:
```suggestion
  mainBranchName: string;
```


I think we should just begin requiring this as a value in the config, though I could be convinced to not require it and have defaults elsewhere.  My concern is about having to constantly manage the default value at the usage site.